src/specs/components/pagination/demo/data.js

```diff
@@ -21,6 +21,20 @@ module.exports = {
         },
       },
     },
+    {
+      aria_label: 'Go to page 1',
```

Comment:
we could also add in the documentation that the first and last pages are always visible